### ISSUE TITLE
Possibility to avoid user failure with loading jsons

### ISSUE THREAD

**Issue by jonathan-dejong** [khromov/acf-local-json-manager#2]
New topic! 

I just noticed that I accidentally saved a field group when _not_ having selected the plugin from which it should be loaded.  That caused a json file to be created in the themes acf-json which then always override the plugins json file when loading the fields.. 

It's a tricky subject because the error is in the usage but perhaps there's something one could do to try to "allow" for these errors to slip by unnoticed. ACF seem to just ignore the later loaded json file (from the plugin) if it's already loaded the themes json file even tho the timestamp is later in the plugins json.

**Comment by khromov** [comment]
I see the problem. Not sure how to solve it from a technical standpoint, but if someone has ideas or a PR they are very welcome!

**Comment by jonathan-dejong** [comment]
Yeah I'm not sure wether I should take this directly to Elliot too and see if he can make sure that timestamps are checked between json files in different locations as well.
